src/components/markdown/CodeBlock/CodeBlock.tsx

```diff
@@ -44,9 +45,11 @@ const LineContent = styled.span`
 `;
 
 const CodeSnippetLineContent = styled(LineContent)`
-  color: #00bb00a0; // any better color?
```

Comment:
`#00bb00a0` isn't an accessible color that only has 4:1 contrast and WCAG AA asks for 4.5:1, so I removed the transparency.